xchange-core/src/test/java/org/knowm/xchange/dto/trade/UserTradeTest.java

```diff
@@ -110,10 +115,10 @@ public void testSerializeDeserialize() throws IOException {
             feeAmount,
             feeCurrency);
 
-    String json = ObjectMapperHelper.toCompactJSON(original);
+    String json = objectMapper.writeValueAsString(original);
```

Comment:
why not just use ObjectMapperHelper.viaJSON ?

Reply:
Done